tests/unit/algos/list/delimiterMaster.js

```diff
@@ -2,12 +2,12 @@
 
 const assert = require('assert');
 const DelimiterMaster =
-    require('../../../../lib/algos/list/delimiterMaster').DelimiterMaster;
+    require('../../../../index').algorithms.list.DelimiterMaster;
```

Comment:
Modifying these lines doesn't make the delimiter unit tests fail but the policy unit tests instead; see [here](http://ci.ironmann.io/gh/scality/Arsenal/829).